package.json

```diff
@@ -16,6 +16,7 @@
   "packageManager": "yarn@1.22.18",
   "scripts": {
     "preinstall": "npx only-allow yarn",
+    "migrate": "nx g @fundamental-ngx/nx-plugin:migrate-to-jest --project=core-action-bar",
```

Comment:
What is this for?

Reply:
Forgot to remove, it was for quickly testing [auto migration generator](https://github.com/SAP/fundamental-ngx/issues/9813#issuecomment-1540738387)